superset-frontend/src/dashboard/components/PropertiesModal.jsx

```diff
@@ -296,8 +296,8 @@ class PropertiesModal extends React.PureComponent {
         footer={
           <>
             <Button
-              type="button"
-              buttonSize="sm"
+              htmlType="button"
```

Comment:
This seems fine, but I'm curious if/why we need it.

Reply:
Me too 😄! I just changed `type` to `htmlType`. I think the developer wanted to make it clear that he's handling the submit.